app/views/homescreen/blocks/_my_account.html.erb

```diff
@@ -0,0 +1,22 @@
+<h3 class="homescreen-blocks--item-header">
+  <%= homescreen_user_avatar %>
+  <%= l(:label_my_account) %>
+</h3>
+
+<ul class="homescreen--arrow-links">
+  <li>
+    <%= link_to l(:label_profile), my_account_path, title: l(:label_profile) %>
+  </li>
+  <li>
+    <%= link_to l(:label_keyboard_shortcut_go_my_page), my_page_path,
+                title: l(:label_keyboard_shortcut_go_my_page) %>
+  </li>
+  <li>
+    <%= link_to l(:button_change_password), my_password_path, title: l(:button_change_password) %>
+  </li>
+  <% if Setting.users_deletable_by_self? %>
+  <li>
+    <%= link_to l('account.delete'), delete_my_account_info_path, title: l('account.delete') %>
+  </li>
+  <% end %>
```

Comment:
I see that the content of this block might change once the "My Account" page was restructured. But do we want to have such a prominent "delete my account" link on the start page?

This is not like a common task you want to reach quickly ^^